src/Bicep.LangServer/Handlers/BicepTelemetryHandler.cs

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Bicep.LanguageServer.Telemetry;
+using MediatR;
+using Newtonsoft.Json.Linq;
+using OmniSharp.Extensions.LanguageServer.Protocol.Models;
+using OmniSharp.Extensions.LanguageServer.Protocol.Workspace;
+
+namespace Bicep.LanguageServer.Handlers
+{
+    public class BicepTelemetryHandler : ExecuteCommandHandler
```

Comment:
Would you mind adding a comment to this class to explain exactly what it's used for, as well as the flow of events between client <-> server?

Reply:
Added comment here:
https://github.com/Azure/bicep/commit/d4161712991bb1c40d64c7f5a0e16e19f0b288b8

Let me know if it sounds good.